semcore/input-tags/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.23.2] - 2024-02-26
+
+### Fixed
+
+- Height for `InputTag` with size `l`.
```

Comment:
Could you please add exact size change in px (like from x px to y px) to let developers expect it?